examples/analysis/ubet/Dockerfile.nitro

```
@@ -0,0 +1,61 @@
+FROM trailofbits/polytracker:latest
+ENV DEBIAN_FRONTEND=noninteractive
+LABEL org.opencontainers.image.authors="kelly.kaoudis@trailofbits.com, henrik.brodin@trailofbits.com"
+WORKDIR /polytracker/the_klondike
+
+# kaoudis, May 2023: Nitro (or more likely, coda-oss) has done something weird
+# and Nitro no longer can find PYTHON_HOME. Using ENABLE_PYTHON=OFF for now
+# so that this Dockerfile at least builds. Enable the Nitro SWIG bindings
+# at your own risk!
+
+RUN apt-get update && \
+	apt-get install -y libcurl4-openssl-dev libssl-dev python3
+
+RUN pip install cxxfilt
+
+RUN git clone https://github.com/mdaus/nitro.git
```

I suggest we pin to the specific version of Nitro that we know works, just in case something upstream breaks our instrumentation later on.

This commit has I chose is from mid-March, but I didn't test it. @kaoudis do you think we need an earlier version?

```suggestion
RUN git clone https://github.com/mdaus/nitro.git
WORKDIR /polytracker/the_klondike/nitro
RUN git checkout 342f552768e249e86df702062ff3f60ea1ec813a
```

---

I think working on a version that matches the version when we were working on this is probably best for now until I/we get a chance to add more analysis stuff here and start working on this again! 

Looks like some recent commits are just merging changes from coda-oss, which may perhaps be used more inside Maxar (Nitro is the only open source use I am currently aware of) and coda-oss does seem to introduce a good chunk of Nitro's variability. I think pinning to b39ccc4c07e84e6c05cecb9ae24143373a3ed8e2 could be better since that would be the one that we were looking at when we did the work. What do you think?

---

Yes! Let's pin the version you are suggesting @kaoudis! That's the one we have a working result from.